pillar/era-articles.yaml

```diff
@@ -19,3 +19,8 @@
     display: https://elife.stencila.io/article-52258/
     # snapshot number missing just for launch
     download: https://hub.stenci.la/api/projects/583/snapshots/13/archive
+
+'43154':
+    date: 2020-10-22T10:30:00Z'
+    display: https://elife.stencila.io/article-43154/v18/
```

Comment:
The other examples don't have the `v18` suffix, is that intentional?

Reply:
Yes, the articles that we launched with didn't use the snapshot version number. But this was an exception for launch and it's needed going forward, so that we point to a specific snapshot.